lib/pong.js

```diff
@@ -32,6 +33,28 @@ var pong = {
     });
   },
 
+  findPlayers: function(user_names, cb) {
+    var findPlayer = function(user_name) {
+      var deferred = Q.defer();
+      pong.findPlayer(user_name, function (err, player) {
+        if (err) {
+          deferred.reject(err);
+        } else {
+          deferred.resolve(player);
+        }
+      });
+      return deferred.promise;
+    };
```

Comment:
This is very nice. I can now write `findPlayers` in 1 line of code.

```javascript
  findPlayer: function (user_name) {
    if (user_name[0] == '@') {
      user_name = user_name.substring(1, user_name.length);
    }
    return Player.where({ user_name: user_name })
      .findOne()
      .then(function (user) {
        var deferred = Q.defer();
        if (user) {
          deferred.resolve(user);
        } else {
          deferred.reject(new Error("User '" + user_name + "' does not exist."));
        }
        return deferred.promise;
      });
  },

  findPlayers: function(user_names) {
    return Q.all(user_names.map(pong.findPlayer));
  },
```

Can it be better?